### ISSUE TITLE
unable to handle large datasets

### ISSUE THREAD

**Issue by victorklos** [vanatteveldt/frogr#1]
When a large (>150) character vector is used as input, an error is thrown:

```
Error in textConnection(output) : all connections are in use
```

In `frog.R` a textConnection is opened (line 50) but it is not closed, causing the number of open file handles to increase and increase.

**Comment by victorklos** [comment]
A workaround is to use

    install_github("victorklos/frogr")

which will be available until the fix is accepted, see pull request #2

**Comment by victorklos** [comment]
Fixed by 0afd07a8
